invokeai/backend/util/logging.py

```diff
@@ -342,8 +342,7 @@ def get_logger(
         cls, name: str = "InvokeAI", config: InvokeAIAppConfig = InvokeAIAppConfig.get_config()
     ) -> logging.Logger:  # noqa D102
         if name in cls.loggers:
-            logger = cls.loggers[name]
-            logger.handlers.clear()
+            return cls.loggers[name]
```

Comment:
nit: Since we're touching this code, it can be tidied up a bit by:
- removing the redundant else-statement, and
- moving `cls.loggers[name] = logger` out of the loop

```python
        if name in cls.loggers:
            return cls.loggers[name]

        logger = logging.getLogger(name)
        logger.setLevel(config.log_level.upper())  # yes, strings work here
        for ch in cls.get_loggers(config):
            logger.addHandler(ch)
        cls.loggers[name] = logger
        return cls.loggers[name]
```

Reply:
Thanks. Fixed.